examples/src/main/python/mllib/naive_bayes_example.py

```diff
@@ -24,16 +24,17 @@
 
 from __future__ import print_function
 
+# $example on$
 import shutil
+# $example off$
 
 from pyspark import SparkContext
 # $example on$
 from pyspark.mllib.classification import NaiveBayes, NaiveBayesModel
 from pyspark.mllib.util import MLUtils
-
-
```

Comment:
If you happen to be not able to build the python doc, I will check tomorrow to help.